docs/src/reference/workflow-config-file.rst

```diff
@@ -983,13 +983,30 @@ columns
 
 frequencies
 -----------
-- Valid attributes:
+-  type: object
+-  description: Parameters for specifying tip frequency calculations via ``augur frequencies``
+-  examples:
+
+.. code:: yaml
+
+   frequencies:
+     pivot_interval_units: "weeks"
+     default:
+       min_date: "6M"
+       narrow_bandwidth: 0.038
+     global_1m:
+       min_date: "1M"
+       narrow_bandwidth: 0.019
+     global_2020_to_2022:
+       min_date: "2020-01-01"
+       max_date: "2022-01-01"
+       narrow_bandwidth: 0.076
+
+Each named traits configuration (``default`` or build-named) supports specification of ``min_date``, ``max_date`` and ``narrow_bandwidth``. Other parameters can only be specified across all builds.
```

Comment:
_(non-blocking)_

Would it be worth allowing build-specific `recent_days_to_censor`, `pivot_interval`, etc. as well? If done right, this could reduce the duplication of functions `_get_min_date_for_frequencies`/`_get_max_date_for_frequencies`/`_get_narrow_bandwidth_for_wildcards`.

Obviously beyond the scope this PR – I could take this or at least make an issue to start it off.

Reply:
Totally. Ideally, every entry in `parameters.yaml` would be able to be over-ridden with build-specific entries in `builds.yaml`. As you say, the current Snakemake workflow strategy here however would require padding out individual functions for each parameter. It would be wonderful to have a generic strategy that makes for automatic over-rides.

I think maybe more important that making this work for `ncov` is thinking through how to do this right across pathogens. Everyone requests the `ncov` style parameter-overrides (this came up for `mpox` most obviously). cc @joverlee521

Reply:
There's definitely been a lot of discussions around build configs outside of `ncov`. I'll take some time to wrangle previous discussions and write up a summary outside of this PR (probably in pathogen-repo-guide).